cli/src/template_builder.rs

```diff
@@ -2647,6 +2671,15 @@ mod tests {
             env.render_ok(r#""bar@other.example.com".remove_suffix("@other.example.com")"#),
             @"bar");
 
+        insta::assert_snapshot!(env.render_ok(r#"" \n \r    \t \r ".trim()"#), @"");
+        insta::assert_snapshot!(env.render_ok(r#"" \n \r testing \t \r ".trim()"#), @"testing");
```

Comment:
nit: here and below, replace the single "testing" by two "foo bar" or something to make sure that we don't remove internal spaces (the implementation is straight-forward so I'm not worried we actually do)